__tests__/utils.tsx

```diff
@@ -89,16 +91,16 @@ describe('getPathInfo()', () => {
 
   beforeEach(() => {
     global.API_ENDPOINT = apiEndpoint
-    mockKV('PATH_INFO_KV', {})
```

Comment:
`mockKV` needs to be still in the tests -> these is not for mocking the fetch function but for mocking the key-value stores we use.